<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HealtCheck</class>
 <widget class="QMainWindow" name="HealtCheck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>478</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>478</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HealtCheck</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>460</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>460</width>
        <height>650</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>460</width>
        <height>650</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_4">
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>10</y>
          <width>231</width>
          <height>91</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 38pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Registery</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>150</y>
          <width>401</width>
          <height>71</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="styleSheet">
            <string notr="true">font: 25pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="text">
            <string>Username</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_username_2">
           <property name="minimumSize">
            <size>
             <width>240</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>240</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>260</y>
          <width>401</width>
          <height>61</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="styleSheet">
            <string notr="true">font: 25pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_password_2">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>240</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>240</width>
             <height>40</height>
            </size>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="pushButton_registery">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>410</y>
          <width>261</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:grey;
font:italic 16pt &quot;Noto Serif Light&quot;;</string>
        </property>
        <property name="text">
         <string>Registery</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_back_to_login">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>520</y>
          <width>280</width>
          <height>50</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>280</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:grey;
font:italic 16pt &quot;Noto Serif Light&quot;;</string>
        </property>
        <property name="text">
         <string>Back to login</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>20</y>
          <width>211</width>
          <height>91</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 38pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>170</y>
          <width>401</width>
          <height>71</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">font: 25pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="text">
            <string>Username</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_username">
           <property name="minimumSize">
            <size>
             <width>240</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>240</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>270</y>
          <width>401</width>
          <height>61</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="styleSheet">
            <string notr="true">font: 25pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_password">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>240</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>240</width>
             <height>40</height>
            </size>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="pushButton_login">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>430</y>
          <width>261</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:grey;
font:italic 16pt &quot;Noto Serif Light&quot;;</string>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>350</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Your login status</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_go_to_registery">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>530</y>
          <width>280</width>
          <height>50</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>280</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:grey;
font:italic 16pt &quot;Noto Serif Light&quot;;</string>
        </property>
        <property name="text">
         <string>Go to registery</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>-20</y>
          <width>451</width>
          <height>71</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font:29pt &quot;MS Shell Dlg 2&quot;;
color:green;
background-color:transparent;</string>
        </property>
        <property name="text">
         <string>Welcome in HealthCheck</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_athlets">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>190</y>
          <width>340</width>
          <height>50</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>340</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:grey;
font:italic 16pt &quot;Noto Serif Light&quot;;</string>
        </property>
        <property name="text">
         <string>Athlet's dietary</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_diets">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>470</y>
          <width>340</width>
          <height>50</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>340</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:grey;
font:italic 16pt &quot;Noto Serif Light&quot;;</string>
        </property>
        <property name="text">
         <string>Diets</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_whr">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>330</y>
          <width>340</width>
          <height>50</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>340</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:grey;
font:italic 16pt &quot;Noto Serif Light&quot;;</string>
        </property>
        <property name="text">
         <string>Dietary calculator (WHR)</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_bmi">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>260</y>
          <width>340</width>
          <height>50</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>340</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:grey;
font:italic 16pt &quot;Noto Serif Light&quot;;</string>
        </property>
        <property name="text">
         <string>Dietary calculator (BMI)</string>
        </property>
       </widget>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>50</y>
          <width>261</width>
          <height>121</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/sport.png);
</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_kcal">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>400</y>
          <width>340</width>
          <height>50</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>340</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:grey;
font:italic 16pt &quot;Noto Serif Light&quot;;</string>
        </property>
        <property name="text">
         <string>Dietary calculator (KCAL)</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_exercises">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>540</y>
          <width>340</width>
          <height>50</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>340</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:grey;
font:italic 16pt &quot;Noto Serif Light&quot;;</string>
        </property>
        <property name="text">
         <string>Exercises</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page">
       <property name="minimumSize">
        <size>
         <width>460</width>
         <height>610</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>460</width>
         <height>610</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(171, 111, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>10</y>
          <width>271</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:black;
font: 30pt &quot;MS Shell Dlg2&quot;;
background-color:transparent;</string>
        </property>
        <property name="text">
         <string>Athlet's dietary</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_back_to_menu">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>530</y>
          <width>280</width>
          <height>50</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>280</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:grey;
font:italic 16pt &quot;Noto Serif Light&quot;;</string>
        </property>
        <property name="text">
         <string>Back to menu</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>100</y>
          <width>431</width>
          <height>401</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>478</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSave">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="separator"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="separator"/>
    <addaction name="actionRedo"/>
   </widget>
   <addaction name="menuSave"/>
   <addaction name="menuEdit"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="Check.qrc">
     <normaloff>:/ch/grafiki/File-New-icon.png</normaloff>:/ch/grafiki/File-New-icon.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Check.qrc">
     <normaloff>:/ch/grafiki/open-file-icon.png</normaloff>:/ch/grafiki/open-file-icon.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Check.qrc">
     <normaloff>:/ch/grafiki/Actions-document-save-icon.png</normaloff>:/ch/grafiki/Actions-document-save-icon.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="Check.qrc">
     <normaloff>:/ch/grafiki/Actions-document-save-as-icon.png</normaloff>:/ch/grafiki/Actions-document-save-as-icon.png</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="Check.qrc">
     <normaloff>:/ch/grafiki/cut-icon.png</normaloff>:/ch/grafiki/cut-icon.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="Check.qrc">
     <normaloff>:/ch/grafiki/Editing-Copy-icon.png</normaloff>:/ch/grafiki/Editing-Copy-icon.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="Check.qrc">
     <normaloff>:/ch/grafiki/Editing-Paste-icon.png</normaloff>:/ch/grafiki/Editing-Paste-icon.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="Check.qrc">
     <normaloff>:/ch/grafiki/Undo-icon.png</normaloff>:/ch/grafiki/Undo-icon.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="Check.qrc">
     <normaloff>:/ch/grafiki/Redo-icon.png</normaloff>:/ch/grafiki/Redo-icon.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Check.qrc"/>
 </resources>
 <connections/>
</ui>
